lib/rubocop/cop/rails/unneeded_receiver_in_with_options.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for unneeded receiver in `with_options`.
```

Comment:
Usually in English the word `redundant` would be used instead of `unneeded`.

Reply:
Thanks for the review. Code comment, cop name, offense message, commit message and so on in this PR were all replaced with `redundant`.